docs/design/schema.md

```diff
@@ -0,0 +1,50 @@
+# Schema
+
+UOR Schema is the attribute type declaration for a UOR Collection. Schema also links application logic to a importing UOR Collection.
+
+This document explains UOR Schema and the relationship of a UOR Collection with its imported Schema. 
+
+## Schema Elements
+
+There are three elements within a schema:
+
+1. Attribute Type Declarations
+2. Algorithm Reference
+3. Default Attribute Mappings
+4. Default Content Reference
+
+### Attribute Type Declarations
+
+Attribute type declarations MUST be written to the manifest annotations of the Schema Collection manifest. Attribute type declarations MUST follow the following syntax and guidelines:
+
+**Key Formatting** - Attribute type declarations expressed via manifest annotation keys MUST follow reverse domain syntax, where the top level domain is the schema short name. 
+
+**Values** - Attribute type declarations expressed via manifest annotation values MUST be one of the following:
+  - a string
+  - a number
+  - a boolean
+  - a dictionary 
+  - null
+
+### Algorithm Reference
+
+Schema Collections MAY contain Algorithm References. The Algorithm Reference in a Schema Collection is the link to the algorithm imported into a calling Collection. This reference is expressed by assigning the `uor.algorithm=true` attribute to the node annotations of the Algorithm's Linked Collection. 
+
+### Default Attribute Mappings
+
+Schema Collections MAY contain Default Attribute Mappings. The Default Attribute Mappings in a schema instruct the UOR Client to add preset attributes to a Collection while being built. This reference is expressed by assigning the `uor.attribute.mapping=true` attribute to the node annotations of the Default Attribute Mappings. 
+
+### Default Content Declaration
+
+Schema Collections MAY contain a Default Content Declaration. The Default Content Declaration in a schema is referenced by an algorithm linked to a collection when the algorithm is rum. This Declaration is expressed by assigning the `uor.dcd={{ dictionary }}` attribute to the Manifest Annotations of the Schema Collection.  
+
+## Design
```

Comment:
Would it be fair to say this could all be expressed in a serialized format like JSON? I believe it was mentioned that this would reside within the manifest or manifest annotations.